spec/support/components/ng_select_autocomplete_helpers.rb

```diff
@@ -6,7 +6,13 @@ def search_autocomplete(element, query:, results_selector: nil)
       element.click
       # Insert the text to find
       within(element) do
-        page.find('input', visible: :all).set(query)
+        input = page.find('input', visible: :all).native
+        input.clear
+        # Send all keys, and then with a delay the last one
+        # to emulate normal typing
+        input.send_keys(query[0..-2])
+        sleep 0.5
+        input.send_keys(query[-1])
```

Comment:
This is a stab at fixing the invite user spec sometimes not triggering the "tag invite" function of ng-select. I couldn't reproduce it outside of tests, and resending parts of the search query fixes it apparently.